adding_species_manuscript.tex

```diff
@@ -536,9 +534,9 @@ \subsection*{Basic setup for chromosome-level simulations}
 These five categories of parameters are sufficient for generating simulations
 under neutral evolution. Such simulations are useful for a number of purposes,
 but they cannot be used to model the influence of natural selection on patterns of genetic variation.
-As mentioned above, it is a widely appreciated fact that linked selection modulates
+As mentioned above, it is a widely appreciated the fact that linked selection modulates
```

Comment:
Original wording seems better to me

Reply:
But is it correct? I think it misses something, but I am not a native speaker :)

Reply:
ah I see, 
```suggestion
As mentioned above, it is widely appreciated the fact that linked selection modulates
```

Reply:
Still wrong IMO :-). 'It is a widely appreciated fact that X'
X is a fact, which is widely appreciated. Seems like a pretty straightforward phrasing. I suggest to keep it as it was, unless others also flag this one.

Reply:
Either the original or the most recent suggestion are grammatically correct in my opinion (though I'd add a comma after "appreciated" in the most recent suggestion), but the original version is smoothest.

Reply:
let's go with the original.